ui/scss/component/_card.scss

```diff
@@ -274,6 +274,23 @@
   display: flex;
   flex-direction: column;
   overflow: hidden;
+
+  .select--slim {
+    //margin-bottom: var(--spacing-xxs);
+    display: flex;
+    //justify-content: center;
+    margin: 0px 0px 10px 10px;
```

Comment:
We should prefer spacing vars instead of px values wherever possible.

Reply:
Hi @jessopb , Alignment fixed. Create a new class for the element.

![image](https://user-images.githubusercontent.com/89370060/187506878-e28a53df-2d21-46eb-ad5a-ace45f039a1e.png)

![image](https://user-images.githubusercontent.com/89370060/187507024-34263797-6b74-4cf3-98a5-0ab430be5fb1.png)

What do you think?

Reply:
Ideally, it's only as wide as the longest server name, but also up to a limit.

Reply:
I think merging comments sounds really interesting, but sounds like a heavy lift, given how it currently works.